Revit_Core_Engine/Modify/Update.cs

```diff
@@ -243,6 +249,91 @@ public static bool Update(this AssemblyInstance element, Assembly bHoMObject, Re
             return true;
         }
 
+        /***************************************************/
+
+        [Description("Updates the existing Revit View based on the given BHoM View.")]
+        [Input("element", "Revit View instance to be updated.")]
+        [Input("bHoMObject", "BHoM View, based on which the Revit View will be updated.")]
+        [Input("settings", "Revit adapter settings to be used while performing the action.")]
+        [Input("setLocationOnUpdate", "Revit View instance does not have location property, therefore this parameter is irrelevant.")]
+        [Output("success", "True if the Revit View instance has been updated successfully based on the input BHoM View.")]
+        public static bool Update<T>(this T element, BH.oM.Adapters.Revit.Elements.View bHoMObject, RevitSettings settings, bool setLocationOnUpdate) where T:View
```

Comment:
I am still not sure why would we need a generic here, couldn't the signature looks as follows:
```suggestion
        public static bool Update(this View element, BH.oM.Adapters.Revit.Elements.View bHoMObject, RevitSettings settings, bool setLocationOnUpdate)
```

Reply:
@pawelbaran, thanks for reflagging this.
I've just forgot it.
Now incorporated in the code.